src/vonage/_internal.py

```diff
@@ -29,3 +31,16 @@ def set_auth_type(client: Client) -> str:
         return 'jwt'
     else:
         return 'header'
+
+
+def validate_phone_number(number: str) -> None:
```

Comment:
Are there unit tests for this method?

Reply:
Good spot! I've added these for the method and also for the auth type check.